src/main/java/org/spongepowered/api/Game.java

```diff
@@ -86,6 +86,13 @@
      */
     Server getServer();
 
+    /**
+     * Gets the {@link Console}
+     *
+     * @return The console
+     */
+    Console getConsole();
```

Comment:
Should bound it to a server as the console ain't really doing anything until the server has started. Also consider preserving integrated server compatibility.

Reply:
https://github.com/SpongePowered/SpongeAPI/commit/a84bf1def1e44936cea81b5a278714fa6f0f9857#r33517014

Reply:
How can you ensure a game only has one console then? At least document its availability (in vanilla, it is only available when a **dedicated** server is present)

Reply:
Does a integrated server have a console directly? Or is it tied to the client (for logging)? A "console" per `Engine` is also an option.

Reply:
> (in vanilla, it is only available when a dedicated server is present)

Yes, in vanilla. We very well could have a console object that is available at game start, rather than server start if we wanted. We might mod the game, but that doesn't mean we need to follow absolutely _everything_ that Mojang does.

I really don't care where this goes and when the console is available, but the thought behind this is that we have a separation of console and server and this is what made sense to me as a result of the conversation there. We might always want a console to exist represent a super user, regardless of whether a physical console exists.

I'm just glad that putting this method here has sparked conversation about it, I just wish I'd put it in a separate PR now and wasn't being so lazy...

Reply:
Ah, didn't notice that console was not for command but is for a more general purpose, mainly logging. I agree with this change then.